app/views/operational_fields/show.html.erb

```diff
@@ -1,28 +1,25 @@
 <% page_title @operational_field.name %>
 <% page_class "operational-field-show" %>
 
-<header class="block headings-block">
-  <div class="inner-block floated-children">
-    <%= render partial: 'shared/heading',
-      locals: { type: 'British fatalities',
-      heading: "Operations in #{@operational_field.name}",
-      big: true, extra: true
-    } %>
-    <div class="heading-extra">
-      <div class="inner-heading">
-        <div class="organisations-icon-list">
-          <%= content_tag_for(:div, @organisation,
-            class: organisation_brand_colour_class(@organisation)) do %>
-            <%= link_to @organisation,
-            class: logo_classes(organisation: @organisation, stacked: true, use_identity: true) do %>
-              <span><%= organisation_logo_name(@organisation) %></span>
-            <% end %>
-          <% end %>
-        </div>
-      </div>
+<div class="govuk-width-container">
```

Comment:
I think this element could still be the template header.

Reply:
Sorry 🙈    what do you mean? 

This element was seemingly not ideal as a `<header>`&  it's not really reflected anywhere else (that I can see on GOV.UK) and having multiple headers in this context when there is a `<header role="banner" id="global-header"` element didn't really align with the use of this element?

Reply:
Doh, you're right we already have a bloody heading! Feel free to resolve this and the other heading comments.